src/Process/Runner/Parallel/ParallelProcessRunner.php

```diff
@@ -51,25 +48,25 @@
  */
 class ParallelProcessRunner
 {
-    private $eventDispatcher;
+    private $processHandler;
 
     /**
-     * @var MutantProcess[]
+     * @var ProcessBearer[]
      */
     private $processesQueue;
 
     /**
-     * @var MutantProcess[]
+     * @var ProcessBearer[]
      */
     private $currentProcesses = [];
 
-    public function __construct(EventDispatcher $eventDispatcher)
+    public function __construct(Closure $processHandler)
     {
-        $this->eventDispatcher = $eventDispatcher;
+        $this->processHandler = $processHandler;
```

Comment:
ParallelProcessRunner does need to know about EventDispatcher, as we may want to reuse ParallelProcessRunner in other places.